nlp/src/main.py

```diff
@@ -77,17 +77,24 @@ async def get_language(request: TextRequest):
 
     results = []
     for text in request.text:
+        languages = []
         prediction = language_model.predict(text, k=2)
-        languages = [x.replace("__label__", "").replace("_Latn", "")
-                          for x in prediction[0]]
-        predictions = ["{:.8f}".format(y) for y in prediction[1]]
+
+        for x in prediction[0]:
+            lang_tag = x.replace("__label__", "").replace("_Latn", "")
+            if tag_is_valid(lang_tag):
+                lang_name = Language.get(lang_tag).display_name("en")
+                languages.append(lang_name)
+            else:
+                languages.append(lang_tag)
+
+        predictions = [f"{y:.8f}" for y in prediction[1]]

```

Comment:
Can we limit it to 2 decimal places? Everything after that is probably just noise.

Reply:
Changed